src/plots/wheelPlot.ts

```diff
@@ -0,0 +1,228 @@
+      return dataToDraw;
+    }
+
+    protected _propertyProjectors(): AttributeToProjector {
+      let attrToProjector = super._propertyProjectors();
+      let innerRadiusAccessor = Plot._scaledAccessor(this.innerRadius());
+      let outerRadiusAccessor = Plot._scaledAccessor(this.outerRadius());
+      let startAngleAccessor = Plot._scaledAccessor(this.startAngle());
+      let endAngleAccessor = Plot._scaledAccessor(this.endAngle());
+      attrToProjector["d"] = (datum: any, index: number, ds: Dataset) => {
+        let startAngle = startAngleAccessor(datum, index, ds);
+        let endAngle = endAngleAccessor(datum, index, ds);
+        if (endAngle < startAngle) {
+          endAngle += (Math.floor((startAngle - endAngle) / (Math.PI * 2)) + 1) * Math.PI * 2;
+        }
+        return d3.svg.arc().innerRadius(innerRadiusAccessor(datum, index, ds))
+                           .outerRadius(outerRadiusAccessor(datum, index, ds))
+                           .startAngle(startAngle)
+                           .endAngle(endAngle)(datum, index);
+      };
+      return attrToProjector;
+    }
+
+    /**
+     * Gets the AccessorScaleBinding for the start angle.
+     */
+    public startAngle<T>(): AccessorScaleBinding<T, number>;
+    /**
+     * Sets the start angle to a constant number or the result of an Accessor<number>.
+     *
+     * @param {number|Accessor<number>} startAngle
+     * @returns {Wheel} The calling Wheel Plot.
+     */
+    public startAngle(startAngle: number | Accessor<number>): Plots.Wheel<R, T>;
+    /**
+     * Sets the start angle to a scaled constant value or scaled result of an Accessor.
+     * This also sets the scale for end angle.
+     * The provided Scale will account for the values when autoDomain()-ing.
+     *
+     * @param {T|Accessor<T>} startAngle
+     * @param {Scale<T, number>} scale
+     * @returns {Wheel} The calling Wheel Plot.
+     */
+    public startAngle<T>(startAngle: T | Accessor<T>, scale: Scale<T, number>): Plots.Wheel<R, T>;
+    public startAngle<T>(startAngle?: number | Accessor<number> | T | Accessor<T>, scale?: Scale<T, number>): any {
+      if (startAngle == null) {
+        return this._propertyBindings.get(Wheel._START_ANGLE_KEY);
+      }
+
+      if (scale != null) {
+        scale.range([0, Math.PI * 2]);
+      }
+
+      let endAngleBinding = this.endAngle();
+      let endAngleAccessor = endAngleBinding && endAngleBinding.accessor;
+      if (endAngleAccessor != null) {
+        this._bindProperty(Wheel._END_ANGLE_KEY, endAngleAccessor, scale);
+      }
+
+      this._bindProperty(Wheel._START_ANGLE_KEY, startAngle, scale);
+      this.render();
+      return this;
+    }
+
+    /**
+     * Gets the AccessorScaleBinding for the end angle.
+     */
+    public endAngle<T>(): AccessorScaleBinding<T, number>;
+    /**
+     * Sets the end angle to a constant number or the result of an Accessor<number>.
+     * The scale of end angle is set to be the scale of start angle.
```

Comment:
We should probably be consistent with the phrasing in `Plots.Rectangle`, which reads
```
If a Scale has been set for X, it will also be used to scale X2.
```

So in this case, "`If a Scale has been set for startAngle, it will also be used to scale endAngle.`".